tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 envlist =
-    py{27,py,34,35,36,37}{,-subunit},coverage,docs
+    py{27,py,35,36,37,38}{,-subunit},coverage,docs
```

Comment:
Was adding py38 here intentional?

Reply:
Yes it was because it is already tested via TravisCI and I'd like to keep tox.ini and .travis.yml in sync.